MagaluCloud/magalu
# fix(network): create proxies to improve cmd tree structure

closes #703



## Description



## Related Issues


## Progress





### Pull request checklist




## How to test it


Run:
```
$ go run . network 
```
check the cmd tree here: https://github.com/profusion/magalu/issues/703


## Visual reference
